components/browser/engine-system/src/main/res/values-uz/strings.xml

```diff
@@ -2,6 +2,9 @@
 <resources>
     <!-- Text for the title of an alert dialog displayed by a web page. %1$s will be replaced with the URL of the current page (displaying the dialog). -->
     <string name="mozac_browser_engine_system_alert_title">%1$s sahifasi xabari:</string>
+    <!-- Text for the message of an auth dialog displayed by a web page.
+    %1$s will be replaced by the realm, %2$s will be replaced with the URL of the current page (displaying the dialog). -->
+    <string name="mozac_browser_engine_system_auth_message">%1$s login va parolingizni soʻrayapti. Sayt xabari: “%1$s”</string>
```

Comment:
>%1$s login va parolingizni soʻrayapti. Sayt xabari: **“%1$s” **

@flodolo @Pike  I think the latter argument should be %2$s?

Reply:
Probably the other way around. I'm going to reject the string in Pontoon for now.